core-bundle/contao/widgets/SerpPreview.php

```diff
@@ -49,25 +50,30 @@ public function generate()
 
 		try
 		{
-			// Get the URL with a %s placeholder for the alias or ID
+			// Get the URL with an %s placeholder for the alias or ID
```

Comment:
```suggestion
			// Get the URL with a %s placeholder for the alias or ID
```

Reply:
PhpStorm made that change. It probably interprets it as _an s placeholder_.

<img width="781" alt="" src="https://github.com/contao/contao/assets/1192057/8e1683dc-f576-46e8-8950-1ab3830f2780">

Reply:
When read out loud you would still say "with a percent-s placeholder", as 's' alone is not the placeholder we are looking for, right? Would PhpStorm still complain if you put it in quotes? i.e. "with a '%s' placeholder"

Reply:
Yes, it would. I consider this a bug in PhpStorm.